fastai/vision/widgets.py

```diff
@@ -17,19 +17,29 @@
 def __getitem__(self:Box, i): return self.children[i]
 
 # Cell
-def widget(im, *args, **layout):
-    "Convert anything that can be `display`ed by IPython into a widget"
+def widget(
+    im, # An object IPython can display to be converted into a widget
+    *args,
+    **layout
+) -> Output: # A context manager to display output as a widget
+    "Convert anything that can be `display`ed by `IPython` into a widget"
```

Comment:
The docments here don't really seem to be adding anything, try again please if not just remove? 😄

Reply:
I removed the docments for now, but maybe "im" should be explained. It is usually an image (hence its name, but more generally it can be anything that can be converted to a widget).